updated index.html

Couch3ater opened the pull request:
also added bootstrap libraries

Couch3ater committed:
made some edits to the web interface

Couch3ater committed:
updated index.html

implemented bootstrap; updated file-input styling; table populates if files present in the /upload/ dir; table appears on page if empty

Couch3ater committed:
added bootstap CSS files

Couch3ater committed:
clenaing codebase

Couch3ater committed:
cleaning code base

updated code to point to CDN's for bootstrap / jquery

Couch3ater committed:
updated git ignore

sadfunction commented on public/index.html:
```diff
@@ -0,0 +1,91 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>fileFU  |  HOME</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.css">
+    <link rel="stylesheet" href="css/style.css">
```
You should add a link to the global.js file here as well. Makes the inline code below body tag extraneous. 

   `<script src="./javascript/global.js"></script>`

Couch3ater replied:
Added ref to script in <head>

sadfunction commented on public/index.html:
```diff
@@ -0,0 +1,91 @@
+        <div class ="col-md-6"><p class="text-center">fileFU is a simple file server brought to you by Dragonegger2.</div>
+      </div>
+
+        <form method="post" action="/api/upload", enctype="multipart/form-data" style="padding-top:10">
```
Going to need id="form" on this tag for the current code to work.

Couch3ater commented on public/index.html:
```diff
@@ -0,0 +1,91 @@
+    </script>
+
+    <!-- The script below updates the file input once a file is selected -->
+    <script>
```
I've moved this into the global.js

Couch3ater commented on public/index.html:
```diff
@@ -0,0 +1,91 @@
+
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+    <script src="/javascript/global.js"></script>
```
Without this ref, the table does not seem to populate; my assumption -- initial ref handles the alert on page load while this ref populates the table once the page has drawn (it's sitting right before the </body>)

Couch3ater committed:
added alerts to state that file has uploaded

table will now repopulate with all files on successful upload; alert will fade out after ~5 seconds; cannot seem to get it to fade out more than once per run-time

Couch3ater committed:
migrated HTML index to JADE
